src/components/Nav/Mobile/Menu.tsx

```diff
@@ -137,15 +137,106 @@ export function Menu() {
 	)
 }
 
+const isYields = (pathname: string) =>
```

Comment:
dont we already have some code that does this? anyway this entire thing is way too manual, it should be changed to something that doesnt require changing it every time something is changed on the menu